americangut/question/ag_question.py

```diff
@@ -0,0 +1,117 @@
+import inspect
+
+import numpy as np
+
+
+class AgQuestion:
+    """A base object class for handling American Gut Data dictionary entries"""
+    true_values = {'yes', 'y', 'true', 1, 1.0, 'Y', 'YES', 'Yes', True}
```

Comment:
no, i mean didn't we aggressively standardize the metadata back in oct/nov? or was that standardization incomplete?